master/internal/core.go

```diff
@@ -820,6 +820,18 @@ func updateClusterHeartbeat(ctx context.Context, db *db.PgDB) {
 	}
 }
 
+func (m *Master) checkIfRMDefaultsAreUnbound(rmConfig *config.ResourceManagerConfig) error {
+	if rmConfig.AgentRM != nil {
+		return m.db.CheckIfRPUnbound(rmConfig.AgentRM.DefaultAuxResourcePool,
+			rmConfig.AgentRM.DefaultComputeResourcePool)
+	}
+	if rmConfig.KubernetesRM != nil {
+		return m.db.CheckIfRPUnbound(rmConfig.KubernetesRM.DefaultAuxResourcePool,
+			rmConfig.KubernetesRM.DefaultComputeResourcePool)
+	}
+	return fmt.Errorf("no Resource Manager found")
```

Comment:
do you have an EE PR ready so we don't break slurm?

Reply:
yeah that's in progress, won't merge until I have EE ready as well

Reply:
https://github.com/determined-ai/determined-ee/pull/1011